node/src/Commands.ts

```diff
@@ -1090,6 +1090,16 @@ export function createZScore(
     return createCommand(RequestType.ZScore, [key, member]);
 }
 
+/**
+ * @internal
+ */
+export function createZMScore(
+    key: string,
+    members: string[],
+): command_request.Command {
+    return createCommand(RequestType.ZMScore, [key, ...members]);
```

Comment:
We should discuss as a team if this is what we want instead of concat. Seems that there are some differences with behaviour and performance. https://www.educative.io/answers/spread-operator-vs-arrayprototypeconcat-in-javascript

Reply:
Good point, from that article it looks like `...` must be used with iterables (true in this case) and has better performance for short lists so maybe that would be best?